spec/models/attendance_spec.rb

```diff
@@ -29,4 +29,16 @@
       expect(attendance.format_date).not_to eq(2.days.ago)
     end
   end
+
+  describe 'human_present' do
+    it 'human present' do
```

Comment:
En general la frase que vaya en el `it` debería leerse o tener un sentido junto a la palabra *it*, algo como `it 'returns present value'` o `it 'returns humanized value for present attribute'`